cmd/wksctl/main.go

```diff
@@ -45,8 +45,12 @@ func configureLogger(cmd *cobra.Command, args []string) {
 }
 
 func main() {
-	clusterv1.AddToScheme(scheme.Scheme)
-	baremetalv1.AddToScheme(scheme.Scheme)
+	if err := clusterv1.AddToScheme(scheme.Scheme); err != nil {
```

Comment:
I'd use `runtimeutil.Must()` here, but no need to change in this PR as I'll fix this up later